contracts/tokenWhitelist.sol

```diff
@@ -241,9 +241,15 @@ contract TokenWhitelist is ENSResolvable, Controllable, Transferrable {
     /// @notice Toggles whether or not a token is redeemable or not.
```

Comment:
There are two `not`s in this sentence which I think is incorrect grammatically.
Also did we stop using @param comments?

Reply:
good catch